DocumentedExamples/Querying_Scalar_Quantities_and_Annually_Averaged_Timeseries.ipynb

```diff
@@ -0,0 +1,2091 @@
+{
```

Comment:
<p>There are a lot of lines on this plot. Perhaps demonstrate with a reduced set?</p><p> </p><p>Also, it would be worth adding a note that these cycle's actually run together. I.e. cycle2 is initialized from the end of cycle1 (with the initialization date reset to 1959). One could demonstrate how to plot the cycles sequentially by manipulating the dates, but that would be some work and I don't think it's neccessary providing the time sequence is explained in words.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I kept only the 6 cycles. This is still lots of lines on this plot, but at least it gets clearer when plotting the annually averaged time series. </p><p> </p><p> About the note, I think it follows one of Angus comment above. I'll add a short description of the experiments when printing the list for the first time.</p>

Reply:
<p>I added a short note when selecting only the 6 cycles from the first 9-experiments list plotted at the beginning of the notebook.</p>